source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,129 @@
 Resolver for Home Networks
+==========================
+
+
+Setting up Unbound
+------------------
+
+Unbound is a powerful validating, recursive, caching DNS resolver. It’s used by some of the biggest tech companies in the world and also by home users who use it for Pi-hole or self-run resolvers. Setting it up for your home network can be quite simple as we’ll showcase below!
+
+Setting up your own DNS resolver for your entire home network requires a couple of things. Namely, a recursive DNS resolver (who knew!), and a dedicated machine where the resolver runs, which is always on and accessible to the entire network. This can be as simple as a Raspberry Pi connected to your home router or any other machine that you is always online!
+
+Because of the variaty of machines that Unbound can run on we cannot create a comprehensive tutorial for all possible options. For the extent of this tutorial we will Ubuntu as a stepping stone which you could use for other machines. For this tutorial Ubuntu 20.04.1 LTS was used.
+
+While you could download the code and build it yourself if wou wanted, it can be as simple as running:
+::
+	apt install unbound
```

Comment:
This document makes no mention of the Unbound version to use. As was noted elsewhere not all distributions have recent versions of Unbound. Should some lower limit on Unbound version be stated in this document?

Reply:
I'll think about this. I assume `apt install` mostly uses the latest version, but maybe not on some systems.